Agents/FeatureInfoAgent/README.md

```diff
@@ -61,29 +61,61 @@ Follow the below configuration steps within the local `queries` directory.
 
 #### Query Restrictions
 
 To properly parse the metadata and timeseries queries, the agent requires the results from queries to fulfill a set format. For each type of query a number of placeholder tokens can be added that will be populated by the agent just before execution. These are:
 
-- `[IRI]`: The IRI of the feature in the request will be injected
+- `[IRI]`: The IRI of the **feature** of interest, i.e. the feature selected within the DTVF (the IRI will be injected here into the request)
 - `[ONTOP]`: The URL of the Ontop service within the stack will be injected
 
-Queries for metadata should not concern themselves with data relating to timeseries (that can be handled within the timeseries query). Queries here need to return a table with two (or optionally three) columns. The first column should be named `Property` and contains the name of the parameter we're reporting, the second should be `Value` and contain the value. The optional third column is `Unit`. These are case sensitive and any other columns will be ignored.
+<ins>Queries for metadata</ins> should not concern themselves with data relating to timeseries (that can be handled within the timeseries query). Queries here need to return a table with two (or optionally three) columns. The first column should be named `Property` and contains the name of the parameter we're reporting, the second should be `Value` and contain the value. The optional third column is `Unit`. **These are case sensitive and any other columns will be ignored**.
 
 
 <p align="center">
     <img src="meta-query-example.jpg" alt="Example result of a metadata query" width="50%"/>
 </p>
 
-Queries for timeseries data need to return the measurement/forecast IRIs (that will be used to grab the actual values from PostGreSQL), as well as parameters associated with each measurement/forecast. Required columns are `Measurement` (or `Forecast`) containing the IRI, `Name` containing a user facing name for this entry, and `Unit` containing the unit (which can be blank). In this case, any other columns reported by the query **will** be picked up and passed back to the visualisation as regular key-value properties.
+<ins>Queries for timeseries</ins> data need to return the measurement/forecast IRIs, i.e. the IRIs which are connected via `ts:hasTimeSeries` to the actual timeseries instances. Those IRIs will be used to grab the actual values from PostGreSQL as well as parameters associated with each measurement/forecast. Required columns are `Measurement` containing the IRI, `Name` containing a user facing name for this entry, and `Unit` containing the unit (which can be blank). In this case, any other columns reported by the query **will** be picked up and passed back to the visualisation as regular key-value properties.
```

Comment:
"In this case, any other columns reported by the query **will** be picked up and passed back to the visualisation as regular key-value properties."

Where would those key-value pairs be displayed? I also tested with the MetOffice data and the "LastUpdated" value for forecasts does not appear in the DTVF despite being returned by the SPARQL query. So, we might need to investigate or update the README accordingly.

![image](https://user-images.githubusercontent.com/79832739/215700372-94093931-8cbe-4e43-b51c-4a73101f187c.png)
![image](https://user-images.githubusercontent.com/79832739/215700785-3cccb791-cac2-4902-9e52-dc9ab80d200f.png)